german/Oktoberfest/oktoberfest.md

```diff
@@ -3,4 +3,6 @@ Sie ist blond und hat lange geflochtene Haare. Als Sie sie anblickt, sie merkt d
 
 [Laden Sie sie zum etwas zu trinken ein](../Einladen/einladen.md)
 
-[Fragen Sie sie "Wie spaet ist es?"](../Fragen/fragen.md)
\ No newline at end of file
+[Fragen Sie sie "Wie spaet ist es?"](../Fragen/fragen.md)
+
+[Heute hab ich einen Anruf aus dem Himmel bekommen, sie vermissen dort einen Engel... Keine Angst ich hab dich nicht verraten! (../Fragen/fragen.md) ]
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.